livekit_internal.proto

```diff
@@ -20,10 +20,23 @@ message NodeStats {
   int64 started_at = 1;
   // when server last reported its status
   int64 updated_at = 2;
-  uint32 num_rooms = 3;
-  uint32 num_clients = 4;
-  uint32 num_tracks_in = 5;
-  uint32 num_tracks_out = 6;
+  // when server previously reported its status
+  int64 updated_at_previous = 7;
```

Comment:
should this be local to the node that is computing the other stats? any value in persisting?